unit/src/MooseUtilsTest.C

```diff
@@ -166,8 +166,14 @@ TEST(MooseUtils, convertStringInt)
   EXPECT_THROW(MooseUtils::convert<long long int>("42.1", true), std::invalid_argument);
 
   EXPECT_EQ(MooseUtils::convert<unsigned long long int>("42"), 42ull);
+  EXPECT_EQ(MooseUtils::convert<unsigned long long int>("18446744073709551615"),
+            18446744073709551615ull);
   EXPECT_EQ(MooseUtils::convert<unsigned long long int>("18446744073709551614"),
             18446744073709551614ull);
+  EXPECT_EQ(MooseUtils::convert<unsigned long long int>("18446744073709551613"),
+            18446744073709551613ull);
+  EXPECT_EQ(MooseUtils::convert<unsigned long long int>("17446744073709551600"),
```

Comment:
Question on this one - just an extra test sufficiently away from max? I wasn't sure if I was missing the significance of this one.

Reply:
Yeah, it is. I just left those in after using them to debug the issue. I could take them back out, but they _don't eat any bread_, as we'd say in Germany...

Reply:
I'm fine with this, especially since you showed that variance in the output on Slack. I'll get to actually testing this after a meeting.